src/main/java/com/microsoft/aad/msal4j/CustomJWTAuthentication.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.aad.msal4j;
```

Comment:
The top of each class should have a comment about copyright and the open-source license we use. You can find it at the top of every file, it's just:
`// Copyright (c) Microsoft Corporation. All rights reserved.`
`// Licensed under the MIT License.`

This should be added to `ConfidentialClientApplicationTest.java` too

Reply:
Sure updated the file. Also, moved tests from ConfidentialClientApplicationTest to an existing file under Integration tests.